extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -213,18 +213,22 @@ export class SubscriptionService {
         }
       }
 
-      this.entitlementStore_.getGrantStatus()
-          .then(grantState => {this.processGrantState_(grantState);});
-
-      this.selectAndActivatePlatform_();
+      this.startUnblockingDocument_();
     });
     return this;
   }
 
   /**
-   * @private
-   * @returns {!Promise}
+   * Unblock document based on grant state and selected platform
    */
+  startUnblockingDocument_() {
```

Comment:
This could also be "startBlocking" - we don't know until we are done. I think this is just `startAuthorizationFlow()` or such.

Reply:
Done